corehq/apps/domain/deletion.py

```diff
@@ -219,6 +223,20 @@ def iter_ids(model_class, field, domain, chunk_size=1000):
         )
 
 
+def _delete_es_docs(es_query, domain_name):
+    query = es_query().domain(domain_name)
+    with silence_during_tests() as stream:
+        doc_ids = with_progress_bar(
+            query.scroll_ids(),
+            query.count(),
+            prefix=es_query.__name__,
+            oneline="concise",
+            stream=stream,
+        )
+        for chunk in chunked(doc_ids, 1000, list):
+            query.adapter.bulk_delete(chunk)
```

Comment:
If deletes fail for some reason, the expectation here is that we will run the deletion script again, right?

Reply:
Correct.